src/main/java/co/cask/PluginJSON.java

```diff
@@ -256,6 +333,14 @@ private void initialize() {
       widgetDirectory = new File(widgetsDirectory);
     }
 
+    if (iconsDirectory != null) {
+      if (!iconsDirectory.contains("/")) {
```

Comment:
this is wrong, it should check that the directory starts with '/', not that it contains '/'.